.github/workflows/build.yml

```diff
@@ -3,14 +3,12 @@ name: Build
 
 on:
   pull_request:
-    push:
-      branches:
-        - main
+  push:
+    branches:
+      - main
```

Comment:
This changes the functionality I think: it'll now also start running on pushes to main. Is that desired?

Reply:
Hm you're right it would now trigger on pushes to `main`, but before it wasn't doing anything. `push` is not a valid key under `pull_request` my github actions linter says. The [documentation ](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request) also seems to back this up.

So I think the `push` section should just be removed.